### DISCUSSION TITLE
Move navidrome datas to a new server

### DISCUSSION THREAD

**Discussion by RickFr410** [navidrome/navidrome#3210] [Q&A] [answered]
Hello,
I try to move navidrome to a new server. I copied all the music files, the playlists and the navidrome.db and navidrome.toml files, but the playlists are imported again  with new ID. What is the good method to do that ?
![image](https://github.com/user-attachments/assets/0f306448-5792-419a-b5b2-8d7825bef682)

Note that I changed the music files path (from /var to /srv). The navidrome.toml file has been updated.

**Comment by deluan** [comment, marked as answer]
Changing the path of the song files cause Navidrome to reimport all as if they were new additions, and remove the old ones. This causes the songs to have new IDs and you lose playcounts, stars and ratings and this also affects the playlists.

This issue will be resolved in #2709. For now, I suggest you restore a backup, keep the same configuration (don't update `navidrome.toml`), and create a symlink to the old path. In this way, the paths are not changed and everything should stay the same.

**Reply by RickFr410** [reply]
Hello,
thanks for the answer, I applied your advise. It's okay.
